content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -83,43 +100,79 @@ Next, configure the :guilabel:`Fiscal Information` in the :guilabel:`Sales & Pur
 - :guilabel:`Commercial Name`: If the company uses a specific commercial name, and it needs to be
   displayed in the invoice.
 
```

Comment:
```suggestion
.. tip::
   The data configured in the :guilabel:`Fiscal Information` section is printed in fiscal valid PDF reports.

.. image:: colombia/fiscal-data-in-fiscal-pdf-con.png
   :alt: Fiscal information printed in PDF.
```
@samueljlieber this is my proposal after deprecating the "Report Configuration" section that was only aimed for Carvajal. (Added the new image to the same [drive folder](https://drive.google.com/drive/folders/11cXF1gXxwws_P9hUslnSJcfjheEbBJZN))